File: Data Structures & Algorithms/concatenation-of-array/submission-1.cpp
class Solution {
   public:
    vector<int> getConcatenation(vector<int>& nums) {
        int n = nums.size();
        vector<int> ans;
        ans.reserve(2*n);
        for(int num : nums) ans.push_back(num);
        for(int num:nums) ans.push_back(num);

        return ans;

    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:

        
        res = {}
        

        for num in nums:
            res[num] = res.get(num, 0) + 1

        ans = sorted(res, key=res.get, reverse=True)[:k]      
        return ans
